i18n.md

```diff
@@ -390,57 +387,3 @@ $ gulp available-languages
 $ git commit client/gettext/po client/gettext/json
 $ git push
 ```
-
-### ManageIQ Amazon Provider
-
-This procedure would apply to all rails engines hooked into ManageIQ in general.
-
-* Update the gettext catalog. With the Amazon provider engine (plugin) hooked into your ManageIQ instance, run the following from the ManageIQ git checkout:
-
-```sh
-$ rake locale:plugin:find[ManageIQ::Providers::Amazon]
-```
-
-* Upload the catalog to zanata:
-
-```sh
-$ zanata-cli push --push-type source
-```
-
-* Once the translations are complete, download the translated catalogs:
-
-```sh
-$ zanata-cli pull --pull-type trans --locales ...
-```
-
-* Commit and push the new catalogs into git:
-
-```sh
-$ git commit locale/
-$ git push
-```
-
-
-## Internationalization in rails engines (plugins)
```

Comment:
I think this part is still available, but optional.

Reply:
@Fryguy I was under the impression that the translations would be happening in https://github.com/ManageIQ/manageiq repo once the plugin is included in the ManageIQ build, and not in individual plugin repos.

Reply:
Yes, I made the change to include strings from all enabled plugins in one ManageIQ resource. And yes, it is also still possible to have a separate catalog for your ManageIQ plugin, but we did that only for plugins that are not enabled & shipped with ManageIQ by default (cloud services & migration analytics). So I'd say it's okay to remove this part from ManageIQ guidelines.